source/adios2/toolkit/sst/CMakeLists.txt

```diff
@@ -28,6 +28,13 @@ if(ADIOS2_SST_HAVE_LIBFABRIC)
   endif()
 endif()
 
+if(ADIOS2_SST_HAVE_UCX)
+  target_sources(sst PRIVATE dp/ucx_dp.c)
+  target_link_libraries(sst
+                  PRIVATE ucx::ucx)
+  set(CMAKE_REQUIRED_INCLUDES ${UCX_INCLUDE_DIRS})
```

Comment:
This should be instead handled by the target when using target_link_libraries

Reply:
Dropped "set(CMAKE_REQUIRED_INCLUDES ${UCX_INCLUDE_DIRS})".